src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/TypeParameterSymbolReferenceFinder.cs

```diff
@@ -27,7 +27,7 @@ protected override bool CanFind(ITypeParameterSymbol symbol)
             // parameter has a different name in different parts that we won't find it.  However,
             // this only happens in error situations.  It is not legal in C# to use a different
             // name for a type parameter in different parts.
-            return FindDocumentsAsync(project, documents, cancellationToken, symbol.Name, symbol.ContainingType.Name);
+            return FindDocumentsAsync(project, documents, SupportsGlobalSuppression(symbol), cancellationToken, symbol.Name, symbol.ContainingType.Name);
```

Comment:
were there type parameter tests?

Reply:
they are not supported in global suppressions